### ISSUE TITLE
Relative timestamp on related content cards fails AA text contrast

### ISSUE THREAD

**Issue by georgeblahblah** [guardian/dotcom-rendering#6133]
The axe extension reports the following for the timestamps on related content cards in dark mode:

> Element has insufficient color contrast of 4.24 (foreground color: #707070, background color: #000000, font size: 11.3pt (15px), font weight: bold). Expected contrast ratio of 4.5:1

We could possibly use [`neutral[60]`](https://github.com/guardian/source/blob/54dd09e234cc5e9392b77bfcb69b09c94fca04e2/packages/%40guardian/source-foundations/src/colour/palette.tsx#L129).

![Screen Shot 2022-10-05 at 12 31 36](https://user-images.githubusercontent.com/705427/194050813-52743371-ad7b-4f9d-9906-e84646fac239.png)
